hystrix-scenario/config/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     links:
       - mock-collector
     volumes:
-     - {AGENT_FILE_PATH}:/usr/local/hystrix-scenario/agent
+      - {AGENT_FILE_PATH}:/usr/local/hystrix-scenario/agent
```

Comment:
Why do you modify this file?

Reply:
this postion is not alignment. so I changed a little. I will revert it